fusor-ember-cli/app/helpers/host-type.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export function hostType(host) {
+    if (this.get('host.is_virtual')) {
+      return new Ember.Handlebars.SafeString('<center><img src="/assets/r/vm-icon-16.png" class="img-responsive"></center>');
+    } else {
+      return new Ember.Handlebars.SafeString('<span class="pficon pficon-screen" style="font-size: 16px"></span>');
```

Comment:
the "bare metal" icon is a patternfly icon, but the "vm icon" is an image that was created by UX team